<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imc</class>
 <widget class="QMainWindow" name="imc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>890</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IMC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>890</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>890</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>731</width>
        <height>884</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>2</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Courier&quot;;
border: none;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Courier'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; color:#afafaf;&quot;&gt;IMC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>12</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>41</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Arial&quot;;
border: none;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:600;&quot;&gt;Cálculo IMC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>81</y>
         <width>731</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">colo: c0c0c0;
font: 10pt &quot;Arial&quot;;
border:none;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;IMC é a sigla para índice de Massa Corpórea, parâmetro adotado pela Organização Mundial de Saúde para calcular o peso ideal de cada pessoa.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;O índice é calculado da seguinte maneira, divide-se o peso do paciente pela sua altura elevada ao quadrado. Diz-se que o indivíduo tem peso normal quando o resultado do IMC está entre 18,5 e 24,9.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Quer descobrir seu IMC? Insira seu peso e sua altura nos campos abaixo e compare com os índices da tabela. Importante, siga os exemplos e use pontos como separadores.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_altura">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>250</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Arial&quot;;
color: #3573c4;</string>
       </property>
       <property name="text">
        <string>Altura</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_170">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>250</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;
color: #3573c4;</string>
       </property>
       <property name="text">
        <string>(ex.: 1.70)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_peso">
       <property name="geometry">
        <rect>
         <x>166</x>
         <y>250</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Arial&quot;;
color: #3573c4;</string>
       </property>
       <property name="text">
        <string>Peso</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_171">
       <property name="geometry">
        <rect>
         <x>201</x>
         <y>250</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Arial&quot;;
color: #3573c4;</string>
       </property>
       <property name="text">
        <string>(ex.: 69.2)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_altura">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>270</y>
         <width>120</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #5D87BC;
border-radius: 8px;</string>
       </property>
       <property name="placeholderText">
        <string>  Metros</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_peso">
       <property name="geometry">
        <rect>
         <x>158</x>
         <y>270</y>
         <width>120</width>
         <height>32</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #5D87BC;
border-radius: 8px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>  Quilos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_calcular">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>315</y>
         <width>115</width>
         <height>40</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1f5eaf;
font: 75 10pt &quot;Arial&quot;;
color: #FFF;
border-radius: 18px;
font-weight: 600;</string>
       </property>
       <property name="text">
        <string>      Calcular</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/a (1).png</normaloff>
         <normalon>../../../../Downloads/a (1).png</normalon>../../../../Downloads/a (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_limpar">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>314</y>
         <width>115</width>
         <height>40</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ff5202;
font: 75 10pt &quot;Arial&quot;;
color: #FFF;
border-radius: 18px;
font-weight: 600;</string>
       </property>
       <property name="text">
        <string>      Limpar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../Downloads/a (2).png</normaloff>
         <normalon>../../../../Downloads/a (2).png</normalon>../../../../Downloads/a (2).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_interpretacao">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>370</y>
         <width>641</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #1f5eaf;
font: 75 10pt &quot;Arial&quot;;
font: 10pt &quot;Arial&quot;;
color: #FFF;
border-radius: 10px;
padding-top: 5px;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;    VEJA A INTERPRETAÇÃO DO IMC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_tabela" native="true">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>422</y>
         <width>691</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_title_imc">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>-1</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 11pt;
color: #3573c4;
font-weight: 500;
border-bottom: 3px solid #e7eaee;</string>
        </property>
        <property name="text">
         <string>   IMC                             CLASSIFICAÇÃO                    OBESIDADE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>555</x>
          <y>10</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 8pt;
color: #3573c4;
font-weight: 500;
</string>
        </property>
        <property name="text">
         <string>(GRAU)</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget_normal" native="true">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>83</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: 3px solid #e7eaee;</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>9</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>ENTRE 18.5 E 24.9</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>222</x>
           <y>10</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>NORMAL</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>521</x>
           <y>10</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
color: #3573c4;
font-weight: 500;
border:none;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_sobrepeso" native="true">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>124</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: 3px solid #e7eaee;</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>6</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>ENTRE 25.0  E 29.9</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>7</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>SOBREPESO</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>521</x>
           <y>9</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;
</string>
         </property>
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_obesidade" native="true">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>165</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: 3px solid #e7eaee;</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>6</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>ENTRE 30.0 E 39.9</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>7</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>OBESIDADE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>518</x>
           <y>9</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;
</string>
         </property>
         <property name="text">
          <string>II</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_obesidade_grave" native="true">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>206</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: 3px solid #e7eaee;</string>
        </property>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>6</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>MAIOR QUE 40.0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>7</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>OBESIDADE GRAVE</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>515</x>
           <y>9</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;
</string>
         </property>
         <property name="text">
          <string>III</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="widget_magreza" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>641</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom: 3px solid #e7eaee;</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>9</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>MENOR QUE 18.5</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>222</x>
           <y>10</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
border: none;
color: #3573c4;
font-weight: 500;</string>
         </property>
         <property name="text">
          <string>MAGREZA</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>521</x>
           <y>10</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 11pt;
color: #3573c4;
font-weight: 500;
border:none;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_seuimc">
       <property name="geometry">
        <rect>
         <x>17</x>
         <y>680</y>
         <width>201</width>
         <height>61</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ff5202;
color: #FFF;
border-radius: 20px;
font-size: 16pt;
font-family: Arial;
padding-left: 10px;
font: bold;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;SEU IMC&lt;span style=&quot; font-size:11pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_imc">
       <property name="geometry">
        <rect>
         <x>126</x>
         <y>687</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ff5202;
color: #FFF;
font-size: 16pt;
font-family: Arial;
font-weight: 700;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
